roles/ceph-mgr/tasks/main.yml

```diff
@@ -5,6 +5,12 @@
   when:
     - containerized_deployment
 
+- name: set a fact for the release number of the current release
```

Comment:
We use this convention for set_fact:

```
- name: set_fact current_ceph_release_num - < explanation >
```

This is helpful when debugging is not turned on.

Reply:
Also I think we want to have this fact in `ceph-common` so the rest of the role will inherit from it (when doing non-container deployments).

Reply:
That makes sense. I wasn't sure if these roles were used independently without `ceph-common`.

I'll go back and add it to `ceph-common` first.